docs/contributing/pull-request-guidelines/index.md

```diff
@@ -41,93 +41,55 @@ The following is a general example of the pull request workflow.
      - If the author does not have write access, ask the reviewers or maintainers.
      - It is the author's responsibility to care about their own pull request until it is merged.
 
-## CI checks
+## Pull request rules
 
-Autoware has several checks for a pull request.
-The results are shown at the bottom of the pull request page as below.
+### Follow `Conventional Commits` (required, automated)
 
-![ci-checks](images/ci-checks.png)
+#### Rationale
 
-If the ❌ mark is shown, click the `Details` button and investigate the failure reason.
+- It can generate categorized changelog, for example using [git-cliff](https://github.com/orhun/git-cliff).
 
-If the `Required` mark is shown, you cannot merge the pull request unless you resolve the error.
-If not, it is optional, but preferably it should be fixed.
+#### Example
 
-The following sections explain about common CI checks in Autoware.  
-Note that some repositories may have different settings.
-
-### DCO
-
-The Developer Certificate of Origin (DCO) is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project.
-
-This workflow checks whether the pull request fulfills `DCO`.  
-You need to confirm the [required items](https://developercertificate.org/) and commit with `git commit -s`.
-
-For more information, refer to the [GitHub App page](https://github.com/apps/dco).
-
-### semantic-pull-request
-
-Autoware uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) with the settings of [commitizen/conventional-commit-types](https://github.com/commitizen/conventional-commit-types).
-
-This workflow checks whether the pull request follows `Conventional Commits`.
-
-Note that if there is only one commit in the pull request, you need to make the commit title semantic as well.  
-This is due to GitHub's non-intuitive behavior when there is only one commit.
-GitHub uses the commit title for the squashed commit message, instead of the pull request title.
-
-For more detailed behaviors of this workflow, refer to [amannn/action-semantic-pull-request](https://github.com/amannn/action-semantic-pull-request).
-
-For the detailed rules, see the [commit guidelines](commit-guidelines.md).
-
-### pre-commit
+```text
+feat(trajectory_follower): add an awesome feature
+```
 
-[pre-commit](https://pre-commit.com/) is a tool to run formatters or linters when you commit.
+Note that you have to start the description part (`add an awesome feature`) with a lowercase.
 
-This workflow checks whether the pull request has no error with `pre-commit`.
+Since Autoware uses the `Squash and merge` method of GitHub, you need to make the PR title follow `Conventional Commits` as well.
```

Comment:
```suggestion
Since Autoware uses the `Squash and merge` method of GitHub, you need to make the PR title follow `Conventional Commits` as well.

It is recommended to have the PR title match the first commit message.
```

Reply:
> It is recommended to have the PR title match the first commit message.

It can be deleted after https://github.com/autowarefoundation/autoware-github-actions/pull/179 is merged.